stanford_help.install

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Install tasks.
+ */
+
+/**
+ * Implements hook_enable().
+ *
+ * This initializes two database variables:
+ *  - settings for this module: stanford_stanford_help_settings
+ *  - configuration for content access: content_access_stanford_help
+ *
+ * For content access configuration, this gives access to view help pages for
+ * 'administrator', 'site owner', and 'editor' roles
+ */
+function stanford_help_enable() {
+
+  // Initialize settings
+  $settings = stanford_help_get_settings();
+  variable_set('stanford_stanford_help_settings', $settings);
+
+  // Set up content access permissions for roles if they exist.
+  // Get role IDs.
+  $roles = array('site owner', 'editor');
+  $caspp = array();
+  $rids = array();
+  foreach ($roles as $role_name) {
+    $role = user_role_load_by_name($role_name);
+    if ($role->rid) {
+      $rids[] = $role->rid;
+    }
+  }
+  if ($admin_role = user_role_load_by_name('administrator')) {
+    $rids[] = $admin_role->rid;
+  }
+
+  // Set the variable for Content Access.
+  $current = variable_get('content_access_stanford_help', array());
+  $caspp = array('view_own' => $rids, 'view' => $rids);
+  $merged = array_merge_recursive_ex($current, $caspp);
```

Comment:
this should be:
$merged = _stanford_help_array_merge_recursive_ex($current, $caspp);